form/form.go

```diff
@@ -0,0 +1,223 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+//Package form enables the completion of forms specified as using environschema.
```

Comment:
// Package form provides ways to fill out forms corresponding to environschema
// schemas.

?